Enums_and_Switches.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+
+
+class Variables(Enum):
+    PAK = "PAKISTAN"
+    IND = "INDIA"
+    AUS = "AUSTRALIA"
+    ENG = "ENGLAND"
+    BAN = "BANGLADESH"
+    NZ = "NEWZEALAND"
+
+
+value = input("Give the abbreviated country name:")
+match value:
```

Comment:
create a function and pass value as enum so you don't have to compare with value